apps/vendor-portal/src/modules/import/composables/useImport/index.ts

```diff
@@ -53,23 +55,38 @@ export default (): IUseImport => {
   const loading = ref(false);
   const uploadedFile = ref<IUploadedFile>();
   const selectedImporter = ref<IImporterMetadata>();
+  const importHistory = ref<ImportPushNotification[]>([]);
 
   //subscribe to pushnotifcation and update the import progress status
   watch(
     () => notifications,
     (newVal) => {
-      if (importStatus.value) {
-        const notification = newVal.value.find(
-          (x) => x.id === importStatus.value.notification.id
-        ) as ImportPushNotification;
-        if (notification) {
-          updateStatus(notification);
-        }
+      const notification = importStatus.value
+        ? (newVal.value.find(
+            (x) => x.id === importStatus.value.notification.id
+          ) as ImportPushNotification)
+        : (newVal.value.find(
```

Comment:
Please add TODO: that it is a temporary workaround to fill history from push notifications and it will be removed when we add support of execution history for import jobs to Api.